northbound/sensorthings/mqtt/src/test/java/org/eclipse/sensinact/gateway/northbound/sensorthings/mqtt/integration/InsecureMqttNotificationsTest.java

```diff
@@ -206,38 +207,42 @@ public void testDatastreamsCollection() throws Exception {
             // by including other resources
             createResource("data", "bar", "foobar", 17);
 
-            List<Datastream> streams = readMessages(6, Datastream.class);
+            List<Datastream> streams = readMessages(8, Datastream.class);
 
             // Creation in sorted event order (p/s/r uri)
-            assertEquals("friendlyName", streams.get(0).name);
-            assertEquals("modelUri", streams.get(1).name);
-            assertEquals("foobar", streams.get(2).name);
+            assertEquals(ProviderPackage.Literals.ADMIN__FRIENDLY_NAME.getName(), streams.get(0).name);
```

Comment:
Again - I'm a bit concerned about using the ProviderPackage here. We want `friendlyName` to be API. If that bit of the model is changed then lots of tests should start failing.

Reply:
If someone would rename the friendlyName attribute to e.g. foo, the Literals would change and ProviderPackage.Literals.ADMIN__FRIENDLY_NAME would not exist anymore. The Literals are like fields in a class and represent the actual Reference in the Model.